Resources/Prototypes/Body/Parts/vox.yml

```diff
@@ -7,7 +7,8 @@
   abstract: true
   components:
   - type: Damageable
-    damageContainer: Biological
+    damageContainer: VoxMade
```

Comment:
`damageContainer`s are used to differentiate between humanoids, walls, and robots, this shouldn't be changed until someone makes a better way to get these differences